pytest_timeouts.py

```diff
@@ -98,6 +85,18 @@ def parse_timeout(timeout):
         timeout = 0.0 if timeout < 0.0 else timeout
         return timeout
 
+    @staticmethod
+    def configure():
+        TimeoutsPlugin.get_markers_func_name = "get_marker"
+        major_ver, minor_ver, _ = pytest.__version__.split('.')
+        try:
+            major_ver = int(major_ver)
+            minor_ver = int(minor_ver)
+            if (major_ver > 3) or ((major_ver == 3) and (minor_ver >= 6)):
+                TimeoutsPlugin.get_markers_func_name = "iter_markers"
+        except ValueError:
```

Comment:
Base on [pep-440](https://www.python.org/dev/peps/pep-0440/#final-releases) release have format ("major.minor") or ("major.minor.micro"), [pytest](https://pypi.org/project/pytest/#history) use second one. When ValueError will be throw?

Reply:
I didn't see declaration in pytest project that it support only this format. Anyway changed